dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/LoginPropertyTest.java

```diff
@@ -0,0 +1,290 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.security.wim.adapter.ldap.fat;
+
+import static componenttest.topology.utils.LDAPFatUtils.assertDNsEqual;
+import static componenttest.topology.utils.LDAPFatUtils.updateConfigDynamically;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.config.wim.AttributesCache;
+import com.ibm.websphere.simplicity.config.wim.LdapCache;
+import com.ibm.websphere.simplicity.config.wim.LdapFilters;
+import com.ibm.websphere.simplicity.config.wim.LdapRegistry;
+import com.ibm.websphere.simplicity.config.wim.LoginProperty;
+import com.ibm.websphere.simplicity.config.wim.SearchResultsCache;
+import com.ibm.websphere.simplicity.log.Log;
+import com.ibm.ws.com.unboundid.InMemoryLDAPServer;
+import com.ibm.ws.security.registry.test.UserRegistryServletConnection;
+import com.unboundid.ldap.sdk.Entry;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+import componenttest.topology.utils.LDAPUtils;
+
+/**
+ * Test the loginProperty. A user should be able to login with any loginProperty -
+ * with or without the userFilter defined.
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class LoginPropertyTest {
+    //Empty server.xml. We create the config here.
+    private static LibertyServer libertyServer = LibertyServerFactory.getLibertyServer("com.ibm.ws.security.wim.adapter.ldap.fat.loginproperty");
+    private static final Class<?> c = LoginPropertyTest.class;
+    private static UserRegistryServletConnection servlet;
+
+    /**
+     * Nearly empty server configuration. This should just contain the feature manager configuration with no
```

Comment:
I would just call this original configuration, and update the comment. This statement was copied from one of my original dynamic tests, and I now regret that I didn't have more generic descriptions.